### ISSUE TITLE
NPE in system tests on windows vista

### ISSUE THREAD

**Issue by rschildmeijer** [rschildmeijer/deft#76]
```
Exception in thread "I/O-LOOP" java.lang.NullPointerException
    at org.deftserver.web.IOLoop.start(IOLoop.java:61)
    at org.deftserver.web.DeftSystemTest$1.run(DeftSystemTest.java:221)
    at java.lang.Thread.run(Thread.java:619)
```

NPE disappears if i remove the following line from our Closeables.closeQuietly
    IOLoop.INSTANCE.removeHandler(channel);

The test that fails is 
    DeftSystemTest.asynchronousRequestTest()

**Comment by rschildmeijer** [comment]
My qualified guess is that this is fixed by commit https://github.com/rschildmeijer/deft/commit/53c711d26282052446e8d2b78c0c8e17d2f1cbb5

**Comment by schildmeijer** [comment]
Fix for #76 (NPE on windows vista) (closed by a875f297cb25a8f023bd9f4361a632575859b06c)
(minor code activation)
